chainer/training/triggers/interval_trigger.py

```diff
@@ -111,3 +111,13 @@ def serialize(self, serializer):
 
     def get_training_length(self):
         return (self.period, self.unit)
+
+    def __str__(self):
+        """Returns a string describing the class and interval
+
+        Returns:
+            str: IntervalTrigger(<period>, '<unit>')
+        """
+        return '%s(%d, \'%s\')' % (
```

Comment:
It seems Chainer uses `format` in other places instead of `%`.

```suggestion
        return '{}({}, \'{}\')'.format(
```